src/common/commitmentTransaction.ts

```
@@ -11,7 +11,7 @@ import type {
   U64,
   UTF8,
 } from "./dataTypes";
-import { arrayCompare, decodeBase58ToFixed, toFixedUint8Array } from "./utils";
+import { arrayCompare, constantTimeEqual, decodeBase58ToFixed, toFixedUint8Array } from "./utils";
```

ESLint

[error] Replace `·arrayCompare,·constantTimeEqual,·decodeBase58ToFixed,·toFixedUint8Array·` with `⏎··arrayCompare,⏎··constantTimeEqual,⏎··decodeBase58ToFixed,⏎··toFixedUint8Array,⏎`

(prettier/prettier)